### PR TITLE
Fix some warnings and errors.

### PR DESCRIPTION
**Pull request by LautaroCesso**

### CODE REVIEW

File: client/src/core-components/tag-selector.scss [hunk 1]
```diff
@@ -18,13 +18,7 @@
     }
   }
 
-  &__tag-options {
-    font-size: 13px;
-    color: $dark-grey;
-  }
-
   &__tag-option {
-    color: $primary-black;
```
Comment (LautaroCesso) [inline client/src/core-components/tag-selector.scss:27]: Before
Screenshot from 2020-06-15 04-23-49

After
Screenshot from 2020-06-15 04-24-14

Comment (LautaroCesso) [inline client/src/core-components/tag-selector.scss:27]: Before
Screenshot from 2020-06-15 04-31-35

After
Screenshot from 2020-06-15 04-31-13

File: client/src/app-components/articles-list.js [hunk 2]
```diff
@@ -89,7 +89,7 @@ class ArticlesList extends React.Component {
 
 export default connect((store) => {
     return {
-        topics: store.articles.topics,
+        topics: store.articles.topics.map((topic) => {return {...topic, private: topic.private === "1" ? true : false}}),
```
Comment (ivandiazwm) [inline client/src/app-components/articles-list.js:92]: just return the condition in the map:
```
topics: store.articles.topics.map((topic) => {return {...topic, private: (topic.private == "1")}}),
```

File: client/src/app/admin/panel/settings/admin-panel-custom-tags-modal.js [hunk 3]
```diff
@@ -103,10 +103,7 @@ class AdminPanelCustomTagsModal extends React.Component {
             }
         }).then(() => {
             this.context.closeModal();
-            this.setState({
-                loading: false,
```
Comment (ivandiazwm) [inline client/src/app/admin/panel/settings/admin-panel-custom-tags-modal.js:107]: why remove this setState

File: client/src/app/admin/panel/staff/admin-panel-departments.js [hunk 4]
```diff
@@ -206,12 +206,13 @@ class AdminPanelDepartments extends React.Component {
...
             form,
             errors,
-            formLoading
-        } = this.state
+            formLoading,
+        } = this.state;
 
         return {
-            values: form,
+            values: {...form, private: form.private ? true : false},
```
Comment (ivandiazwm) [inline client/src/app/admin/panel/staff/admin-panel-departments.js:213]: image

File: client/src/app/admin/panel/staff/admin-panel-view-staff.js [hunk 5]
```diff
@@ -32,7 +32,14 @@ class AdminPanelViewStaff extends React.Component {
     }
 
     getProps() {
-        return _.extend({}, this.state.userData, {
+        const { userData } = this.state;
+        const userDataWithNumericLevel = {
+            ...userData,
+            level: userData.level*1,
+            sendEmailOnNewTicket: userData.sendEmailOnNewTicket === "1" ? true : false
```
Comment (ivandiazwm) [inline client/src/app/admin/panel/staff/admin-panel-view-staff.js:39]: use only `userData.sendEmailOnNewTicket === "1"`

File: client/src/app/admin/panel/tickets/admin-panel-all-tickets.js [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+            showDepartmentDropdown: false,
+            departments,
+            tickets,
+            type: 'secondary',
+            loading,
+            ticketPath: '/admin/panel/tickets/view-ticket/',
+            onPageChange: this.onPageChange.bind(this),
+            page,
+            pages,
+            closedTicketsShown: closedTicketsShown ? true : false,
```
Comment (ivandiazwm) [inline client/src/app/admin/panel/tickets/admin-panel-all-tickets.js:77]: use `!!`

**Pull request merged**